### DISCUSSION TITLE
Specify template for the message

### DISCUSSION THREAD

**Discussion by deepaksharma-507** [dcblogdev/laravel-microsoft-graph#51] [Q&A]
Hi, 

I am in the process of replacing the available SMTP functionality provided by Laravel framework, with MSGraph, to send emails. With the current method I am able to pass a template, in this case a blade view. Does anything such exists within this package ? Am I able to specify templates for my email body? If yes, then how? 

Thanks,
Deepak

**Comment by dcblogdev** [comment]
not currently, this package will do that https://github.com/InnoGE/laravel-msgraph-mail
